python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -110,6 +122,8 @@ def to_json(self) -> Dict:
                 feature if isinstance(feature, str) else feature.to_json()
                 for feature in self.features
             ],
+            "sliding_window_timestamp_field": self.timestamp_field,
```

Comment:
There is no longer any field named `sliding_window_timestamp_field` in this class.